app/assets/stylesheets/components/_post.scss

```diff
@@ -80,6 +83,64 @@ $Post-inlineCodeBgColor: rgba(0, 0, 0, 0.04);
     }
   }
 
+  .Table {
```

Comment:
Yes, but that's by design. That rule should not have any knowledge that `Table` even exists. Instead, it should referencing a specific descendant like `Post-table` for instance.